source/common/grpc/google_grpc_utils.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <cstdint>
+#include <string>
+
+#include "envoy/buffer/buffer.h"
+
+#include "grpcpp/grpcpp.h"
+
+namespace Envoy {
+namespace Grpc {
+
+class GoogleGrpcUtils {
+public:
+  /**
+   * Build grpc::ByteBuffer which aliases the data in a Buffer::InstancePtr.
+   * @param bufferInstance source data container.
+   * @return byteBuffer target container aliased to the data in Buffer::Instance and owning the
+   * Buffer::Instance.
+   */
+  static grpc::ByteBuffer makeByteBuffer(Buffer::InstancePtr&& bufferInstance);
+
+  /**
+   * Build Buffer::Instance which aliases the data in a grpc::ByteBuffer.
+   * @param byteBuffer source data container.
```

Comment:
Nit: rename the variable to be `byte_buffer`, and also the comment. Same for above method.

Reply:
done